code/game/atoms_movable.dm

```diff
@@ -201,13 +184,15 @@
 	if(pulledby && !pulledby.pulling)

```

Comment:
Doubt we need `!pulledby.pulling` anymore to prevent that one exploit; not positive; could test. Doubt it really hurts anything either.